jb/itch5/order_book_depth.hpp

```diff
@@ -153,9 +177,10 @@ class order_book {
 
   buys buy_;
   sells sell_;
+  book_depth_t book_depth_ = 0;       
```

Comment:
It is constant time, so is {return buy_.size() + sell_.size(); };
For vector, set, map, queue:
 size_type
  size() const
  { return this->_M_impl._M_finish - this->_M_impl._M_start; } 

...Instead of updating the variable (++ , and --) every time it changes (besides, compute_inside does not use it... so why does it need to keep it?). 
I will remove it!